src/generator/CodeGenerator.php

```diff
@@ -26,20 +26,26 @@ public function __construct($config = null) {
 			$this->config = $config;
 		} else {
 			$this->config = new CodeGeneratorConfig();
-		} 
+		}
 
 		$this->init();
 	}
-	
+
 	protected function init() {
 		if ($this->config->getGenerateEmptyDocblock()) {
```

Comment:
Passing many parameters to visitors will become messy over time. I'd rather pass the `Code(File)GeneratorConfig` instead - what you think?